proj/proj.mk

```diff
@@ -318,6 +318,11 @@ load: $(SOFTWARE_BIN)
 # Load hook allows common_soc.py to provide board-specific changes to load.
 # This isn't ideal, the logic is starting to get too voluminous for a Makefile.
 	$(SOC_MK) load_hook
+	@sleep 2
```

Comment:
Don't use sleep like this.
Use something like;
```shell
while [ ! -f $(TTY) ]; do echo "Waiting for UART"; sleep 1; done
```

Reply:
Thanks, works like a charm.